apisix/admin/init.lua

```diff
@@ -19,6 +19,7 @@ local core = require("apisix.core")
 local route = require("apisix.utils.router")
 local plugin = require("apisix.plugin")
 local v3_adapter = require("apisix.admin.v3_adapter")
+local filter = require("apisix.router").filter
```

Comment:
We need to export filter first so we can use it:
https://github.com/apache/apisix/blob/ae400b95b91240fb867c3cedc9fa65a6b5593f36/apisix/router.lua#L30